storybook/manager.js

```diff
@@ -13,7 +13,9 @@ import { addons } from '@storybook/addons';
 import yourTheme from './theme.js';
 
 addons.setConfig({
+    // `theme` is deprecated in favor of `system` but maintaining `theme` as a deprecated path.
     theme: yourTheme,
+    system: yourTheme,
```

Comment:
I'm pretty sure this is Storybook API and not ours...please confirm.

Reply:
Yes! These are predefined APIs from the `addons.setConfig` method in storybook.
https://storybook.js.org/docs/addons/addons-api#addonssetconfigconfig